CONTRIBUTING.md

```diff
@@ -118,30 +118,46 @@ Once you are satisfied with your PR you should ask someone to review it. Before
-
-In this branch, you should normally only update the version number in the `src/multi-vector-simulator/version.py` and in the `CHANGELOG.md` files, respecting the indicated formats. Commit the first one with "Bump version number" as commit message.
-
-Your `CHANGELOG.md` file could look like this before the release
-```
-## [unreleased]
-
-### Added
-- feature 1
-- feature 2
-### Changed 
-- thing 1
-- thing 2
-### Removed
-- some stuff
-```
-
-Simply replace `unreleased` by `X.Y.Z` and add the date of release in [ISO format](https://xkcd.com/1179/), then add the structure for a new `unreleased` version
-
+    For meaning of X, Y and Z version numbers, please refer to this [semantic versioning guidelines](https://semver.org/spec/v2.0.0.html).
+2. In your release branch, update the version number in [`src/multi-vector-simulator/version.py`](https://github.com/rl-institut/multi-vector-simulator/blob/dev/src/multi_vector_simulator/version.py) (commit message: "Bump version number") and
+in [`CHANGELOG.md`](https://github.com/rl-institut/multi-vector-simulator/blob/dev/CHANGELOG.md) in the in 1. indicated format.
+3. Add the date of release in the `Changelog.md` in [ISO format](https://xkcd.com/1179/).
+4. After pushing these changes, create a pull request from `release/vX.Y.Z` towards `master` and merge it into `master`.
+
+### The actual release
+1. Open a working python3 virtual environment and make sure you have the latest versions of setuptools and wheel installed:
+`python3 -m pip install --user --upgrade setuptools wheel twine`
+2. Delete `build/` and `dist/` directory to avoid errors.
+3. Make sure you are in `master` and the latest commit you want to release and create the package:
+`python3 setup.py sdist bdist_wheel`
+4. Check the package with twine: `python3 -m twine check dist/*` If errors occur, fix them before the release or postpone.
+5. Create a [release tag](https://github.com/rl-institut/multi-vector-simulator/releases) on github.
+Please choose `master` as target and use `vX.Y.Z` as tag version. In the description field simply copy-paste the content of the `CHANGELOG`descriptions for this release and you're done!
+For help look into the [github release description](https://help.github.com/en/github/administering-a-repository/creating-releases).
+6. Pull `master` to your local repository and check if the new tag is there. (If you do not see the tag type `git fetch`.)
+7. Test your package:
+    1. Test the upload on test.pypi.org
+        ` twine upload --repository testpypi dist/*`
+    2. Test the installation: `pip install -i https://test.pypi.org/simple/ multi-vector-simulator --no-deps`
+    3. Then open a python console and try
+        `>>>from multi_vector_simulator import cli`
+8. If everything works as expected you can now upload the package version to PyPI
+    1. Check the credentials of our pypi@rl-institut.de account on https://pypi.org.
+    1. Type `twine upload dist/*`
```

Comment:
```suggestion
    2. Type `twine upload dist/*`
```

Reply:
Funny fact: This would still be shown as "1., 2. etc." on readthedocs no matter which number is written. But still we can get it right in the md-file.